controllers/util/k8sutil.go

```diff
@@ -52,8 +52,8 @@ const (
 	EnableCephfsKey             = "ROOK_CSI_ENABLE_CEPHFS"
 
 	// This is the name for the FieldIndex
-	OwnerUIDIndexName   = "ownerUID"
-	AnnotationIndexName = "annotation"
+	OwnerUIDIndexName   = "index:ownerUID"
+	AnnotationIndexName = "index:annotation"
```

Comment:
Any reason to include this in this PR?

Reply:
https://github.com/red-hat-storage/ocs-operator/pull/3075#discussion_r2011379661 but will remove

Reply:
removed.